test-end-to-end/webapp-cypress/cypress/e2e/WEB_007_001_admin_enable_or_disable_user_registration.cy.js

```diff
@@ -4,10 +4,36 @@ describe('WEB_007_001_admin_enable_or_disable_user_registration', () => {
     cy.ustadStartTestServer(6000)
   })
 
-it('Admin enable registration', () => {
+it('Enable registration switch test', () => {
   // Admin user login
   cy.ustadClearDbAndLogin('admin','testpass',{timeout:8000})
-  cy.ustadEnableUserRegistration()
+  cy.get('#settings_button').click()
+  cy.contains('Site').click()
+  cy.contains('Edit').click()
+  cy.contains('Guest login enabled').should('be.visible')
+  cy.contains('Terms and policies').should('be.visible')
+  cy.contains('Registration allowed').should('be.visible')
+  cy.wait(500) // Without wait switch not clickable
+  cy.get('#registration_allowed').click({force:true})
```

Comment:
No workarounds should be needed here. We should not be using force=true. 
If one removes force=true, and then removes wait, component should work as expected.